pkg/tasks/tasks.go

```diff
@@ -45,11 +44,20 @@ func (tl *TaskRunner) RunAll() error {
 			klog.V(3).Infof("running task %d of %d: %v", i+1, len(tl.tasks), ts.Name)
 			err := tl.ExecuteTask(ts)
 			klog.V(3).Infof("ran task %d of %d: %v", i+1, len(tl.tasks), ts.Name)
-			return errors.Wrapf(err, "running task %v failed", ts.Name)
+			return taskErr{error: errors.Wrapf(err, "running task %v failed", ts.Name), name: ts.Name}
 		})
 	}
+	if err := g.Wait(); err != nil {
+		taskName := ""
+		if tErr, ok := err.(taskErr); ok {
+			taskName = tErr.name
+			err = tErr.error
 
-	return g.Wait()
+		}
+		klog.Info(taskName)
```

Comment:
nit: is this a debug leftover? If not let's consolidate that log line, as the caller has one too:
https://github.com/openshift/cluster-monitoring-operator/blob/2bbe87e182cc8815a60bcd004b6851e30ad0e85c/pkg/operator/operator.go#L317

Reply:
No just a debug msg, sorry about that. We already include the task name that failed in the error few lines before that in the task.go, so this line should print it out.

Removed. PTAL